.web5-spec/openapi.d.ts

```diff
@@ -0,0 +1,498 @@
+/**
+ * This file was auto-generated by openapi-typescript.
+ * Do not make direct changes to the file.
+ */
+
+
+export interface paths {
```

Comment:
Do we expect corresponding test vectors for all of these methods?

Reply:
Eventually yes. This is an auto-generated file